dagger.json

```diff
@@ -2,5 +2,5 @@
   "name": "chainloop",
   "sdk": "go",
   "source": "extras/dagger",
-  "engineVersion": "v0.12.3"
+  "engineVersion": "v0.13.0"
```

Comment:
my local dagger version is 0.13, so I decided to bump it. The changes in the go.mod seems to also be a cause of `go develop`

Reply:
They released the 0.13 last week, we should all update to latest.